squirrel.frontier/src/main/java/org/dice_research/squirrel/frontier/recrawling/SparqlhostConnector.java

```diff
@@ -0,0 +1,117 @@
+package org.dice_research.squirrel.frontier.recrawling;
+
+import org.aksw.jena_sparql_api.core.QueryExecutionFactory;
+import org.aksw.jena_sparql_api.core.UpdateExecutionFactory;
+import org.aksw.jena_sparql_api.core.UpdateExecutionFactoryHttp;
+import org.aksw.jena_sparql_api.http.QueryExecutionFactoryHttp;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.Credentials;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.impl.client.AbstractHttpClient;
+import org.apache.http.protocol.HttpContext;
+import org.apache.jena.atlas.web.auth.HttpAuthenticator;
+import org.apache.jena.query.Query;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QuerySolution;
+import org.apache.jena.query.ResultSet;
+import org.apache.jena.rdf.model.RDFNode;
+import org.apache.jena.sparql.core.DatasetDescription;
+import org.dice_research.squirrel.data.uri.CrawleableUri;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SparqlhostConnector implements OutDatedUriRetreiver {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SparqlhostConnector.class);
+
+    /**
+     * The Query factory used to query the SPARQL endpoint.
+     */
+    protected static QueryExecutionFactory queryExecFactory = null;
+    protected UpdateExecutionFactory updateExecFactory = null;
```

Comment:
Why do you have the UpdateExecutionFactory here? It is not used so why should it be an attribute?